specs/dwritecore/DWriteCore.md

```diff
@@ -1235,3 +2025,906 @@ DWRITE_BEGIN_INTERFACE(IDWriteFactory8, "202D44F0-84AC-4D6F-B8D0-704B27675CAA")
+            /// Center Y coordinate.
+            /// </summary>
+            float centerY;
+
+            /// <summary>
+            /// Start of the angular range of the gradient, measured in counter-clockwise degrees
+            /// from the direction of the positive x axis.
+            /// </summary>
+            float startAngle;
+
+            /// <summary>
+            /// End of the angular range of the gradient, measured in counter-clockwise degrees
+            /// from the direction of the positive x axis.
+            /// </summary>
+            float endAngle;
+        } sweepGradient;
+
+        /// <summary>
+        /// Valid for paint elements of type DWRITE_PAINT_TYPE_GLYPH.
+        /// Specifies a glyph shape to be filled or, equivalently, a clip region.
+        /// This paint element has one child element.
+        /// </summary>
+        /// <remarks>
+        /// The child paint element defines how the glyph shape is filled. The child element can be a single paint
+        /// element, such as a linear gradient. Or the child element can be the root of a visual tree to be rendered
+        /// with the glyph shape as a clip region.
+        /// This corresponds to a PaintGlyph record in the OpenType COLR table.
+        /// </remarks>
+        struct PAINT_GLYPH
+        {
+            /// <summary>
+            /// Glyph index of the glyph that defines the shape to be filled.
+            /// </summary>
+            UINT32 glyphIndex;
+        } glyph;
+
+        /// <summary>
+        /// Valid for paint elements of type DWRITE_PAINT_TYPE_COLOR_GLYPH.
+        /// Specifies another color glyph, used as a reusable component.
+        /// This paint element has one child element, which is the root paint element of the specified color glyph.
+        /// </summary>
+        /// <remarks>
+        /// This corresponds to a PaintColorGlyph record in the OpenType COLR table.
+        /// </remarks>
+        struct PAINT_COLOR_GLYPH
+        {
+            /// <summary>
+            /// Glyph index of the referenced color glyph.
+            /// </summary>
+            UINT32 glyphIndex;
+
+            /// <summary>
+            /// Clip box of the referenced color glyph, in ems. This is an empty rectangle of the color glyph does
+            /// not specify a clip box. If it is not an empty rect, the client is required to clip the child content
+            /// to this box.
+            /// </summary>
+            D2D_RECT_F clipBox;
+        } colorGlyph;
+
+        /// <summary>
+        /// Valid for paint elements of type DWRITE_PAINT_TYPE_TRANSFORM.
+        /// Specifies an affine transform to be applied to child content.
+        /// This paint element has one child element, which is the transformed content.
+        /// </summary>
+        /// <remarks>
+        /// This corresponds to paint formats 12 through 31 in the OpenType COLR table.
+        /// </remarks>
+        typedef DWRITE_MATRIX PAINT_TRANSFORM;
```

Comment:
Another case where this typedef doesn't really seem like it's worthwhile.

Reply:
Agreed.